app/_data/docs_nav_kuma_dev.yml

```diff
@@ -19,13 +19,95 @@ items:
             url: "/introduction/how-kuma-works/#vm-and-k8s-support"
           - text: Kuma vs XYZ
             url: "/introduction/how-kuma-works/#kuma-vs-xyz"
-      - text: Deployments
-        url: /introduction/deployments/
       - text: Enterprise
         url: /introduction/enterprise/
       - text: Changelog
         absolute_url: true
         url: https://github.com/kumahq/kuma/blob/master/CHANGELOG.md
+  - title: Kuma in Production
+    group: true
+    items:
+      - title: Deployment topologies
+        group: true
+        items:
+          - text: Overview
+            url: /production/deployment/
+          - text: Standalone deployment
+            url: /production/deployment/stand-alone/
+          - text: Multi-zone deployment
+            url: /production/deployment/multi-zone/
+          - text: Default ports
+            url: /production/deployment/networking/
+      - title: Control plane deployment
+        group: true
+        items:
+          # - text: Kong Mesh license
+            # url: /production/
+          - text: Deploy a multi-zone global control plane
+            url: /production/cp-deployment/multi-zone/ 
+            group: true
+            items:
```

Comment:
@fabianrbz Do you know why the subnav items below "Deploy a multi-zone control plane" won't show on the site?

Reply:
I don't think we support so many nesting levels, we could add that though
You could add `title: Deploy a multi-zone global control plane` and they should show up but the navbar won't work properly